src/main/resources/templates/errors/error-403.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html lang="en"
+      xmlns:th="http://www.thymeleaf.org">
+<head>
+    <meta charset="UTF-8">
+    <title>403 Error</title>
+</head>
+<body>
+
+<h1 th:text="${errorTitle}"></h1>
+<hr>
+
+<th:block th:if="${errorMessage == null}">
+    <h3>You're not authorized!</h3>
+</th:block>
+<th:block th:if="${errorMessage != null}">
+    <h3 th:text="${errorMessage}"></h3>
+</th:block>
+<p>Please <a th:href="@{/login}">SIGN IN</a> or <a th:href="@{/register}">REGISTER</a></p>
```

Comment:
<p>Please <a th:href="@{/login}">SIGN IN</a> or <a th:href="@{/register}">REGISTER</a></p> this has nothing to do with 403 errors. 
status code 403 - INTERNAL SERVER ERROR triggers when there is a problem on the server side and has nothing to do with user authentication/ authorization